app/src/main/java/com/codesoom/assignment/errors/ProductNotFoundException.java

```diff
@@ -0,0 +1,11 @@
+package com.codesoom.assignment.errors;
+
+/**
+ * 주어진 id의 상품을 찾지 못하면 예외를 던진다.
+ */
+public class ProductNotFoundException extends RuntimeException{
```

Comment:
이 예외 클래스의 코드를 잘 읽어보면 어디에도 `id`에 대한 이야기는 없습니다.
이렇게 작성해보세요.
```suggestion
/**
 * 상품을 찾지 못했을 때 던집니다.
 */
public class ProductNotFoundException extends RuntimeException{
```

Reply:
생성자 파라미터만 변경하고 주석을 신경 못섰나보네요.
감사합니다.